eng/templates/stages/deploy.yaml

```diff
@@ -203,6 +203,24 @@ stages:
         .\darc\darc.exe get-default-channels --source-repo arcade-services --ci -t "$(GetAuthInfo.FederatedToken)" --bar-uri "$(GetAuthInfo.BarUri)" --debug
       displayName: Test Federated token authentication
 
+    - ${{ if in(variables['Build.SourceBranch'], 'refs/heads/main', 'refs/heads/production') }}:
+      - powershell: 
+          $darcBuild = .\darc\darc.exe get-build `
+            --repo "https://github.com/dotnet/arcade-services" `
+            --commit $(Build.SourceVersion) `
+            --ci `
+            -t "$(GetAuthInfo.FederatedToken)" `
+            --output-format json |
+            ConvertFrom-Json
+
+          .\darc\darc.exe add-build-to-channel `
+            --id $darcBuild[0].id `
+            --channel "General Testing"
+            --ci `
+            -t "$(GetAuthInfo.FederatedToken)" `
+            --azdev-pat $(dn-bot-dnceng-build-rw-code-rw-release-rw) `
+        displayName: Test Darc
```

Comment:
Nit: this is technically testing the command specifically, not just Darc in general

Reply:
updated